rpcserver_test.go

```diff
@@ -411,3 +412,86 @@ func TestHandleTestMempoolAcceptFees(t *testing.T) {
 		})
 	}
 }
+
+// TestGetTxSpendingPrevOut checks that handleGetTxSpendingPrevOut handles the
+// cmd as expected.
+func TestGetTxSpendingPrevOut(t *testing.T) {
+	t.Parallel()
+
+	require := require.New(t)
+
+	// Create a mock mempool.
+	mm := &mempool.MockTxMempool{}
+	defer mm.AssertExpectations(t)
```

Comment:
👍 

started to use this defer pattern elsewhere to ensure that the expectations are actually validated